### PR TITLE
Add relations to table

### PR DESCRIPTION
**Pull request by Ngufor-emmanuella**
### **In this miles stone of the project I did the following**

1. Use primary key & foreign key mechanism for joining tables.
2. Query multiple tables.
3. Updated various tables created
4. Use the JOIN to query the data
_Attached below are screenshots taken.
Screenshot (82)
Screenshot (83)
Screenshot (84)
Screenshot (85)
Screenshot (86)
Screenshot (87)
Screenshot (88)
Screenshot (89)
Screenshot (90)
Screenshot (91)
Screenshot (92)
Screenshot (93)
_

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -11,4 +11,24 @@ CREATE TABLE animals (
```
Comment (1GORDON) [inline schema.sql:11]: Great job @Ngufor-emmanuella 🤝

- [ ] As stated in the project requirements, kindly add a new table with the name `vets`.
- [ ] As stated in the project requirements, kindly add a new join table with the name `specializations`.
- [ ] As stated in the project requirements, kindly add a new join table with the name `visits`

File: data.sql [hunk 2]
```diff
@@ -12,4 +12,27 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (' Ditto', date '2022-05-14', 4, true, 22);
+
+-- project for multiple tables., inserting data in owners table 
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34);
+INSERT INTO owners (full_name, age) VALUES ('Jennifer Orwell', 19);
+INSERT INTO owners (full_name, age) VALUES ('Bob', 45);
+INSERT INTO owners (full_name, age) VALUES ('Melody Pond', 77);
+INSERT INTO owners (full_name, age) VALUES ('Dean Winchester', 14);
+INSERT INTO owners (full_name, age) VALUES ('Jodie Whittaker', 38);
+
```
Comment (1GORDON) [inline data.sql:24]: Great job @Ngufor-emmanuella 🤝

- [ ] As stated in the project requirements, kindly insert all the data for `vets`, `specializations`, and `visits` into the data.sql file.

File: queries.sql [hunk 3]
```diff
@@ -49,3 +49,20 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
```
Comment (1GORDON) [inline queries.sql:52]: Great job @Ngufor-emmanuella 🤝

- [ ] As stated in the project requirements, kindly add all the queries into the queries.sql file

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Ngufor-emmanuella ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions

- [ ] Kindly fix the broken MIT license link to make your readme file more professional.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 2 above)
Comment (Ngufor-emmanuella) [reply data.sql:24]: thank you sorry i submitted for the wrong project.this project work is for day three which is Vet clinic database: query multiple tables thanks

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
### Hi @Ngufor-emmanuella, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ngufor-emmanuella** [comment]
thank you

**Pull request merged**